_maps/map_files/PubbyStation/PubbyStation.dmm

```diff
@@ -30496,8 +30496,8 @@
 /area/medical/medbay/zone2)
 "bzF" = (
 /obj/machinery/door/window/eastleft{
-	name = "Monkey Pen";
-	req_one_access_txt = "5"
+	name = "Petting Garden";
```

Comment:
👀

Reply:
![image](https://user-images.githubusercontent.com/33292112/73420152-d7b27500-4321-11ea-8d46-dd1fa30f8c6c.png)

Reply:
cute

Reply:
blame the genetics move :slightly_smiling_face: